source/arriving/setup_computer.rst

```diff
@@ -155,7 +155,46 @@ TODO
 Nextflow
 """"""""
 
-TODO
+Nextflow is an open-source pipelining tool that makes processing massive computational workflows somewhat easy. We use it in the lab to run our various :ref:`ref_flow`. 
```

Comment:
The part "processing massive computational workflows" sound weird... we process data but can we really process workflows?

Reply:
I changed it, is it better?